workflows/preprocessing.nf

```diff
@@ -134,6 +135,34 @@ workflow PREPROCESSING {
     }
     .set {ch_input_fastq}
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// STEP: UMI CONSENSUS (optional, before alignment)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+UMI_FLAG = Channel.value(params.umi_in_readname as boolean)
+if (params.enable_umi) {
+
+    // Convert to [meta, r1, r2] for the UMI subworkflow
+    CH_UMI_FASTQ = ch_input_fastq.map { meta, reads ->
```

Comment:
Can you make the variable name lowercase here? Uppercase names are used for subworkflow and processes